### ISSUE TITLE
rustic with rclone 1.61.0 doesn't work

### ISSUE THREAD

**Issue by cameronbraid** [rustic-rs/rustic#352]
error is 

```
Error: url must start with http://! url: [http://127.0.0.1:40417/]
```

using rclone 1.59 works fine
its serve output is 
```
2022/12/21 02:12:21 NOTICE: S3 bucket octane-site-staging: Serving restic REST API on http://127.0.0.1:46095/
```

where as 1.61 is 
```
2022/12/21 02:14:01 NOTICE: S3 bucket octane-site-staging: Serving restic REST API on [http://127.0.0.1:38023/]
```

**Comment by aawsome** [comment]
Thanks a lot @cameronbraid for the report!
I'll adapt the url parsing to also support 1.61.

**Comment by aawsome** [comment]
#353 should fix this. Can you try the PR?

**Comment by aawsome** [comment]
I tried rclone `1.61` myself. There seems to be more changes to the `rclone serve restic` command.
The treatment of the changed log output is fixed in #353.

Seems to be that also issue https://github.com/rclone/rclone/issues/6644 is a bug which prevents to use rclone 1.61 even with #353 applied.
 I'll merge #353 as it does what it should but suggest to wait for rclone 1.61.1.

**Comment by aawsome** [comment]
I'll re-open this issue until the rclone bug is fixed.

**Comment by cameronbraid** [comment]
thanks

**Comment by aawsome** [comment]
This has been fixed in rclone 1.61.1, see the link to the rclone issue above.
